src/lib/viewers/text/CSVViewer.js

```diff
@@ -53,38 +52,31 @@ class CSVViewer extends TextBaseViewer {
     load() {
         super.load();
 
-        const { representation, location } = this.options;
+        const { representation } = this.options;
         const template = representation.content.url_template;
-        const assetUrlCreator = createAssetUrlCreator(location);
-        const papaWorkerUrl = assetUrlCreator(JS[0]);
 
         return Promise.all([this.loadAssets(JS), this.getRepStatus().getPromise()])
             .then(() => {
-                this.api.get(papaWorkerUrl, { type: 'blob' }).then(papaWorkerBlob => {
-                    /* global Papa */
-                    const workerSrc = URL.createObjectURL(papaWorkerBlob);
-                    Papa.SCRIPT_PATH = workerSrc;
-
-                    this.startLoadTimer();
-                    const urlWithAuth = this.createContentUrlWithAuthParams(template);
-                    Papa.parse(urlWithAuth, {
-                        download: true,
-                        error: (err, file, inputElem, reason) => {
-                            const error = new PreviewError(ERROR_CODE.LOAD_CSV, __('error_refresh'), { reason });
-                            this.handleDownloadError(error, urlWithAuth);
-                        },
-                        complete: results => {
-                            if (this.isDestroyed() || !results) {
-                                return;
-                            }
-
-                            this.checkForParseErrors(results);
-
-                            this.data = results.data;
-                            this.finishLoading();
-                            URL.revokeObjectURL(workerSrc);
-                        },
-                    });
+                this.startLoadTimer();
+                const urlWithAuth = this.createContentUrlWithAuthParams(template);
+                Papa.parse(urlWithAuth, {
+                    delimitersToGuess: [',', '\t'],
+                    download: true,
+                    error: (err, file, inputElem, reason) => {
```

Comment:
Did we validate that these parameters are still accurate/available with the new version?

Reply:
Yeah, the new version still follows the same callback signature